node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -532,6 +534,12 @@ class NodeRuntime private[node] (
     _ <- handleUnrecoverableErrors(program)
   } yield ()
 
+  def setupNodeProgramF[F[_]: LastApprovedBlock: Metrics: Span: BlockStore: ConnectionsCell: NodeDiscovery: TransportLayer: RPConfAsk: SafetyOracle: LastFinalizedBlockCalculator: Sync: Concurrent: Time: Log: EventLog: BlockDagStorage: EngineCell: EnvVars: RaiseIOError: RuntimeManager: Running.RequestedBlocks](
+      casperInit: CasperInit[F]
+  ) =
+    for {
```

Comment:
This for is unnecessary

Reply:
agreed, but at the same time next commits will push more stuff here.